pkg/config/controller/config.go

```diff
@@ -69,6 +69,8 @@ type ControllerConfig struct {
 	IPsecCSRSignerConfig IPsecCSRSignerConfig `yaml:"ipsecCSRSigner"`
 	// Multicluster configuration options.
 	Multicluster MulticlusterConfig `yaml:"multicluster,omitempty"`
+	// AuthenticationOption provides different options for client auth when accessing antrea-controller's api-server.
```

Comment:
I think it should be "API server" or "apiserver"?

Reply:
updated with "apiserver"